src/libOpenImageIO/exif.cpp

```diff
@@ -280,8 +280,14 @@ class TagMap {
         for (int i = 0;  tag_table[i].tifftag >= 0;  ++i) {
             const EXIF_tag_info *eti = &tag_table[i];
             m_tagmap[eti->tifftag] = eti;
-            if (eti->name)
-                m_namemap[eti->name] = eti;
```

Comment:
I'm not sure I like this solution. It feels like something fishy is going on here. What is the value of `eti->name` when it crashes?